src/main/java/server/usecases/create_gc/CreateGCInteractor.java

```diff
@@ -46,8 +46,10 @@ public void create(CreateGCInputData input_data) {
         }
         GroupChat gc = new GroupChat(this.database.getNewUID(), admin, members);
         database.addGC(gc);
+        int peerPort = database.getPeerPortFromUID(input_data.getAdmin());
+        String peerID = database.getPeerIDFromUID(input_data.getAdmin());
         CreateGCOutputData output = new CreateGCOutputData(
-                input_data.getAdmin(), input_data.getMembers(), input_data.getTime());
+                input_data.getAdmin(), input_data.getMembers(), peerID, peerPort);
```

Comment:
What about the name of the groupchat? A chat has uid, name description, I think at least the uid needs to be sent to the clients

Reply:
Also, you also need to send the groupchat info to other members of the group chat, not just the admin, for example, a user can input `$createchat <chat name> <member uid1> <member uid2> ....` and when a group chat is created the information of the chat will be sent to all members including the admin (I assume the admin is also the person who created the chat?)